components/board/genericlinux/board.go

```diff
@@ -477,4 +477,9 @@ func (b *sysfsBoard) Close() {
 	b.cancelFunc()
 	b.mu.Unlock()
 	b.activeBackgroundWorkers.Wait()
+	if !b.usePeriphGpio {
+		if err := gpioCloseAll(); err != nil {
```

Comment:
[minor] This `Close` function should change to return the error from trying to close the pins. Our components are all closed using the `TryClose` method from our utils repo, which handles the case of the `io.Closer` interface as well and seeks to return the error, so it would be good to let higher level logic handle the error appropriately